src/AutoRest.CSharp/Common/Output/Expressions/System/SystemExtensibleSnippets.SystemJsonElementSnippets.cs

```diff
@@ -1,25 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System.ClientModel.Internal;
-using AutoRest.CSharp.Common.Output.Expressions.KnownValueExpressions;
-using AutoRest.CSharp.Common.Output.Expressions.Statements;
-using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
-using AutoRest.CSharp.Common.Output.Models;
-
 namespace AutoRest.CSharp.Common.Output.Expressions.System
 {
     internal partial class SystemExtensibleSnippets
     {
         private class SystemJsonElementSnippets : JsonElementSnippets
         {
-            public override ValueExpression GetBytesFromBase64(JsonElementExpression element, string? format) => InvokeExtension(typeof(ModelSerializationExtensions), element, nameof(ModelSerializationExtensions.GetBytesFromBase64), Snippets.Literal(format));
```

Comment:
is this class even needed any longer?

Reply:
maybe this is no longer needed because Azure version and System version now has no difference.
I will remove it in this PR or later PRs